### PR TITLE
fix(front-end): Доработки

### PR DESCRIPTION
**Pull request by dontuse**
https://jira.railsc.ru/browse/GOODS-589

### CODE REVIEW

File: src/ImageEditor/e-image-editor.scss [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 .e-image-editor {
   & img { max-width: 100%; max-height: 100%; }
+  & p { margin: 0 }
```
Comment (apress-hound) [inline src/ImageEditor/e-image-editor.scss]: Declaration should be terminated by a semicolon

File: src/Dialog/rc-dialog.scss [hunk 2]
```diff
@@ -51,8 +51,8 @@
     & #{$root}-body { background: #f4f4f4; margin-bottom: 0; padding-top: 20px; }
     & #{$root}-footer { background: #f4f4f4; }
     & #{$root}-title { font-weight: normal; font-size: 12px; line-height: 1.67; color: #1f2532; }
-    & h3 { margin: 0; line-height: 1.42; font-family: OpenSans; font-size: 24px; font-weight: 600;}
-
+    & h3 { margin: 0 0 18px; line-height: 1.42; font-family: OpenSans; font-size: 24px; font-weight: 600;}
+    & #{$root}-header { padding-bottom: 17px }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Declaration should be terminated by a semicolon

File: dist/assets-help.bundle.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "20041f0dca40dc9e0d36"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "d0bd261d6d8c07396864"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

**Comment by dontuse** [comment]
@Kupov

**Pull request merged**